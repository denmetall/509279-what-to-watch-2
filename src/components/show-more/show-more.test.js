import React from 'react';
import ShowMore from './show-more.jsx';
import renderer from 'react-test-renderer';

it(`Components ShowMore renders correctly`, () => {
  const tree = renderer
    .create(<ShowMore/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
