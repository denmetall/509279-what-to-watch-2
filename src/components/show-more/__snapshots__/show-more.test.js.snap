// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components ShowMore renders correctly 1`] = `
<button
  className="catalog__button"
  type="button"
>
  Show more
</button>
`;
